common/src/main/java/com/microsoft/identity/common/internal/cache/AccountCredentialCache.java

```diff
@@ -204,38 +170,38 @@ public synchronized Credential getCredential(final String cacheKey) {
             }
         }
 
-        Logger.exiting(TAG, methodName, accounts);
+        Logger.verbose(TAG, "Returning [" + accounts.size() + "] Accounts w/ keys...");
 
         return accounts;
     }
 
     @Override
+    @NonNull
     public synchronized List<Account> getAccounts() {
-        final String methodName = "getAccounts";
-        Logger.entering(TAG, methodName);
-
+        Logger.verbose(TAG, "Loading Accounts...(no arg)");
         final Map<String, Account> allAccounts = getAccountsWithKeys();
         final List<Account> accounts = new ArrayList<>(allAccounts.values());
-
-        Logger.exiting(TAG, methodName, accounts);
-
+        Logger.info(TAG, "Found [" + accounts.size() + "] Accounts...");
         return accounts;
     }
 
     @Override
+    @NonNull
     public List<Account> getAccounts(
-            final @Nullable String homeAccountId,
-            final String environment,
-            final @Nullable String realm) {
-        final String methodName = "getAccounts";
-        Logger.entering(TAG, methodName, homeAccountId, environment, realm);
-
+            @Nullable final String homeAccountId,
+            @NonNull final String environment,
+            @Nullable final String realm) {
+        Logger.verbose(TAG, "Loading Accounts...");
         if (StringExtensions.isNullOrBlank(environment)) {
             throw new IllegalArgumentException("Param [environment] cannot be null.");
         }
 
         final boolean mustMatchOnHomeAccountId = !StringExtensions.isNullOrBlank(homeAccountId);
         final boolean mustMatchOnRealm = !StringExtensions.isNullOrBlank(realm);
+
+        Logger.verbose(TAG, "mustMatchOnHomeAccountId? [" + mustMatchOnHomeAccountId + "]");
```

Comment:
I'm not sure this is going to be clear when reading the logs.... you may just want to log where you're only matching based on homeAccountId or on homeAccountId + Tenant Id.